crates/teloxide/src/update_listeners/webhooks.rs

```diff
@@ -23,6 +23,13 @@ pub struct Options {
     /// [addr]: (self::Options.address)
     pub url: url::Url,
 
+    /// Server-internal path that Axum will listen for requests on.
```

Comment:
This shouldn't mention Axum IMO, these options might be usable for webhooks implemented with other servers.